frontend/test/lit/test_device_api.py

```diff
@@ -90,8 +91,9 @@ def test_preprocess():
     using the new device API.
     TODO: we need to readd the two check-not once we accept the device preprocessing."""
 
-    # CHECK:    quantum.device["[[PATH:.*]]librtd_lightning.{{so|dylib}}", "dummy.remote", "{'shots': 2048}"]
-    dev = DummyDevice(wires=2, shots=2048)
+    # CHECK
+    # quantum.device["[[PATH:.*]]librtd_null_device.{{so|dylib}}", "NullDevice", "{'shots': 2048}"]
```

Comment:
Same here: it should be one line.

```suggestion
    # CHECK:    quantum.device["[[PATH:.*]]librtd_null_device.{{so|dylib}}", "NullDevice", "{'shots': 2048}"]
```